contentcuration/contentcuration/frontend/channelEdit/utils.js

```diff
@@ -178,6 +238,36 @@ export function secondsToHms(seconds) {
   return hms;
 }
 
+/**
+ * Set a default completion model for audiovisual content
+ * equal to the file length in seconds (parsed), or a fallback placeholder
+ *
+ * @param {Object} node
+ * @returns {String|Number} {from options in the map above}
+ */
```

Comment:
Non-blocking. What is meant by "from options in the map above"? Could you rather describe that it returns human-readable duration or "-"?